lib/puma/binder.rb

```diff
@@ -48,7 +48,6 @@ def initialize(log_writer, conf = Configuration.new)
 
       @envs = {}
       @ios = []
-      localhost_authority
```

Comment:
Intentional delete?

Reply:
Yes.  It's called early in the [ssl section](https://github.com/puma/puma/blob/ae81065a0cf7c893a9c60088e71e87685f5c029a/lib/puma/binder.rb#L229) of `parse`.  The call in the above section happens in `initialize`, so it's called regardless of whether an ssl bind is being used.